src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2138,6 +2138,7 @@ private BoundExpression BindAddressOfExpression(PrefixUnaryExpressionSyntax node
         // Basically a port of ExpressionBinder::isFixedExpression, which basically implements spec section 18.3.
         // Renamed because there are already too many things called "fixed".
         // NOTE: internal purely for testing purposes.
+        // CONSIDER: why is it used in product code? also, accessedLocalOrParameterOpt is not used anywhere!
```

Comment:
>also, accessedLocalOrParameterOpt is not used anywhere! [](start = 53, length = 55)

It looks like the NonMoveableVariables_Dereference unit-test demonstrates that the symbol is actually used.